admin/database/postgres/migrations/0006.sql

```diff
@@ -1,3 +1,4 @@
+ALTER TABLE projects ADD COLUMN subpath TEXT NOT NULL;
```

Comment:
You can't edit an existing migration file, even if it was just added. You have to move to a new one